WalletWasabi.Fluent/Views/MainWindow.axaml

```diff
@@ -20,7 +20,7 @@
         ExtendClientAreaChromeHints="Default, PreferSystemChrome, OSXThickTitleBar"
         Title="Wasabi Wallet"
         WindowState="{Binding WindowState, Mode=TwoWay}"
-        Icon="/Assets/WasabiLogo.ico">
+        Icon="avares://WalletWasabi.Fluent.Desktop/Assets/WasabiLogo.ico">
```

Comment:
Is this a good idea?
I mean it gets the icon from a `Desktop` project, it will definitely look weird when we will have the `Mobile` project too.
Maybe it is not an issue but looked not ok at the first glance, and needs to discuss. Maybe it has a consequence for the future.

Reply:
We need to discues

> Is there any reference on the .png icons at all? Couldn't we delete them?

I'm afraid we can't. They're needed by the **Packager** here
https://github.com/zkSNACKs/WalletWasabi/blob/880690e4adb0f795653d7740a0c124cef210615d/WalletWasabi.Packager/Program.cs#L430

Deleting them will cause this issue https://github.com/zkSNACKs/WalletWasabi/issues/6217

Reply:
> Is this a good idea? I mean it gets the icon from a `Desktop` project, it will definitely look weird when we will have the `Mobile` project too. Maybe it is not an issue but looked not ok at the first glance, and needs to discuss. Maybe it has a consequence for the future.

I'm agree.

I think we need to discuss. This is just a workaround, IMO.

The Packager will look in the Desktop project path for WasabiLogo*.png, but Views and ViewModels inside WalletWasabi.Fluent, should not depend on assets located in the Desktop project.

I'd do the following:

- Move shared assets (including) .png files to Fluent
- Remap paths in Packager and the application (will no longer use `avares://WalletWasabi.Fluent.Desktop`)
- Modify Fluent.Desktop cproj to take the application icon from WalletWasabi.Fluent

Also, I'd do this **as part of another PR**.

What do you think?

cc @zkSNACKs/visual-design-group

Reply:
> Remap paths in Packager and the application (will no longer use avares://WalletWasabi.Fluent.Desktop)

I think the Packager is ok to get the files from the Desktop project as it creates packages for desktops.
I am not sure. Maybe that is why the file was duplicated, which is not perfect yeah, but it caused less confusion.
Honestly, I am not fully familiar with this or how this could be properly fixed, without file duplication.

Reply:
I'll try to make this without duplication and a bit better design-wise. Later we can decide if it's OK or not. What do you think? :)

Reply:
I've tried to improve, but I'm afraid there's no good way to avoid duplicating the .ico file.

I've tried with a shortcut, but CSC cannot locate the file correctly. 
The easy fix is to keep this little duplication. It's a very small tradeoff.